chrome/content/zotero/preferences/preferences_export.xul

```diff
@@ -49,10 +50,15 @@
 			<label value="&zotero.preferences.quickCopy.defaultOutputFormat;" control="quickCopy-menu"/>
 			<menulist id="zotero-quickCopy-menu"/>
 			
-			<separator class="thin"/>
-			
-			<checkbox id="zotero-quickCopy-copyAsHTML" label="&zotero.preferences.quickCopy.copyAsHTML;"
-				oncommand="Zotero_Preferences.Export.buildQuickCopyFormatDropDown(document.getElementById('zotero-quickCopy-menu'), this.checked ? 'html' : '');"/>
+			<hbox align="center">
+				<label value="&zotero.preferences.quickCopy.locale;" control="locale-menu"/>
+				<menulist id="locale-menu"/>
```

Comment:
https://github.com/rmzelle/zotero/blob/locale-selector/chrome/content/zotero/integration/integrationDocPrefs.xul has `onload="Zotero_File_Interface_Bibliography.init();"`, and the `init` function calls on the "locale-list" menulist.